lib/tasks/foreman_theme_satellite_tasks.rake

```diff
@@ -44,13 +44,19 @@ namespace :foreman_theme_satellite do
 
     all_links = ForemanThemeSatellite::Documentation::USER_GUIDE_DICTIONARY
                 .merge(ForemanThemeSatellite::Documentation::PLUGINS_DOCUMENTATION)
-                .merge(ForemanThemeSatellite::Documentation.flat_docs_guides_links)
 
     failed = all_links.filter { |_key, doc_address| doc_address.include?('/html/') && !checker.test_link(doc_address) }
 
+    DOCS_GUIDE_CHAPTER_MAPPING.each do |guide, chapters|
+      chapters.keys.each do |chapter|
+        url = ForemanThemeSatellite::Documentation.docs_url(guide, chapter)
+        failed["/#{guide}/#{chapter}"] = url if url.include?('/html-single/')
```

Comment:
Not sure if this reports correctly.

Reply:
The `html-single` code path is the unrecognized guide. I admit it's very indirect. Note that it can only be reached if it's not found in `DOCS_GUIDE_PAGES` (which is the TOC).

Reply:
Let me rephrase: If I have a link without a chapter: just for the guide. It will not get into the `if chapter` block in the `docs_url` method, and will return `<root>/non-existing-guide`. Which is an error too, if I read the code correctly.

Reply:
There we assume `guide.downcase` returns the correct result, but you're right that we actually have a list of guides and can check. And I'll go through the list of current guides and add mappings

Reply:
I updated the mapping based on https://github.com/theforeman/foreman-documentation/blob/master/guides/upstream_filename_to_satellite_link.json but when verifying I found some mismatches: https://github.com/theforeman/foreman-documentation/pull/3181